diagnostic.md

```diff
@@ -36,19 +36,22 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
+The path listed above is a relative path because it tells you exactly where to go from the Home directory. It gives a direct route, avoiding online URLs that may change over time and therefore lose the information they linked to.
+
+
 
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+We would use an absoulte path because that's where the font is being hosted by the server. It similar to the argument made in class: any photo hosted by google isnt going anywhere, we'll be able to link to it as long is its being hosted.
  <!-- Answer Ends Here -->
```

Comment:
Correct. Just to be clear, the font is _a_ server - likely the server of the website that is providing this font - not the server of the user.